docker/pypi/wmagent/bin/manage

```diff
@@ -158,10 +163,69 @@ init_wmagent(){
 }
 
 
+##############################
+#  Tier0 stuff               #
+##############################
+
+
+# generate the tier0 config from the basic template
+init_tier0(){
+    local errVal=0
+    wmagent-mod-config --input=$WMA_CONFIG_DIR/config-template.py \
+                       --output=$WMA_CONFIG_DIR/config-agent.py \
+                       --coredb_url=oracle://$ORACLE_USER:$ORACLE_PASS@$ORACLE_TNS \
+                       --working_dir=$WMA_INSTALL_DIR \
+                       --couch_url=http://$COUCH_USER:$COUCH_PASS@$COUCH_HOST:$COUCH_PORT \
+                       --workqueue_db_name=$LOCAL_WORKQUEUE_DBNAME \
```

Comment:
For my education, some of these env vars are extracted by WMAgent secrets, right?

Reply:
Anything starting with `WMA_*`  is coming from the environment, the rest from  WMAgent.secrets

Reply:
And the origin of the full set of variables used in the current script is explained here: https://github.com/dmwm/CMSKubernetes/blob/e2d764a0a4c970b3ceff58f913e9bdee10328b25/docker/pypi/wmagent/bin/manage#L11-L58